### DISCUSSION TITLE
Breadcrumbs show file name

### DISCUSSION THREAD

**Discussion by Saorsa32** [chrisgurney/obsidian-note-toolbar#59] [Ideas]
If I am using {{prop.up}} is there a way to show the note's title instead of just a word or icon?

**Comment by chrisgurney** [comment]
@Saorsa32 Thanks for the idea! 

Not at the moment, but I can take a look into it. I agree that it would be helpful!

**Comment by chrisgurney** [comment]
@Saorsa32 Try out [version 1.8.3](https://github.com/chrisgurney/obsidian-note-toolbar/releases/tag/1.8.3) which now lets you use variables in labels and tooltips. Let me know how it works for you!
